benchmarks/implementations/joint/bindings/python/Component.py

```diff
@@ -1,5 +1,6 @@
 import pyjoint
 from .Benchmarks_adapters import *
+from builtins import range
```

Comment:
Add future:builtins to REQUIRED_PYMODULES in scripts/check_dependencies.sh

Reply:
done